good_first_issues/graphql/services.py

```diff
@@ -142,6 +162,16 @@ def identify_mode(name: str, repo: str, user: bool) -> Tuple[str, Dict, str]:
         variables["name"] = name
         mode = "org"
 
+    if hacktoberfest and not repo:
+        query = search_query
+        variables["queryString"] = 'topic:hacktoberfest '
+        if name:
+            variables["queryString"]+=f'user:{name}'
+        mode="search"
```

Comment:
We can name it to `hacktober` for easy understanding.

Reply:
@yankeexe I have named the variable as ``queryString`` so that it is common and the same query(search_query) can be reused when we implement features like search by language or any other criteria.

Reply:
sure, makes sense now. 💯